http-client/ingredients-database_utils.ts

```diff
@@ -0,0 +1,72 @@
+import {renderProduct} from '../products_utils'
+import {ingredientData} from '../type/ingredient-data'
```

Comment:
this is invalid import, add index.ts and export things.
Imports should be one level deep eg.:
```
import { ingredientData } from '../type'
```

Also file name should be probably `types`